### PR TITLE
fix: 'py_glob' referenced before assignment

### PR DESCRIPTION
**Pull request by delgermurun**

### CODE REVIEW

File: normalizer/core.py [hunk 1]
```diff
@@ -353,6 +353,7 @@ def normalize(
     test_glob = list(work_path.glob('tests/test_*.py'))
 
     class_name = None
+    py_glob = []
```
Comment (JoanFM) [inline normalizer/core.py:356]: remove line 370

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Comment by mapleeit** [comment]
Please also adopt this change to make test run. 
https://github.com/jina-ai/jina/pull/4830/files

**Comment by delgermurun** [comment]
@mapleeit there is `protobuf` in requirements. Where should I put it?

**Comment by mapleeit** [comment]
I think we should add it.

**Comment by delgermurun** [comment]
added

**Review by mapleeit — APPROVED** [review]
LGTM

**Review by JoanFM — APPROVED** [review]

**Pull request merged**